components/playlist/resources/components/app.tsx

```diff
@@ -100,7 +105,8 @@ export class PlaylistPage extends React.Component<Props, State> {
   }
 
   onClickRemoveItemButton = (playlistItemId: string) => {
-    getPlaylistAPI().removeItemFromPlaylist('', playlistItemId)
+    getPlaylistAPI().removeItemFromPlaylist(
```

Comment:
Can the `currentList` ever be `null | undefined`? I guess maybe if you remove all the playlists, but is that something we let the user do?